app/services/hyrax/solr_service.rb

```diff
@@ -111,6 +111,20 @@ def query(query, **args)
       end
     end
 
+    def query_in_batches(query, **args)
```

Comment:
written this by hand a bunch of times to do stuff in the console. finally found a reason to make it something permanent.